homeassistant/components/garmin_connect/sensor.py

```diff
@@ -70,12 +64,12 @@ async def async_setup_entry(
     async_add_entities(entities, True)
```

Comment:
```suggestion
    async_add_entities(entities)
```

Reply:
Ah yes that was it, I should have remembered that...